scss/civicrm/dashboard/pages/_view-dashboard.scss

```diff
@@ -0,0 +1,233 @@
+.column {
+  .widget-controls {
+    padding: 17px 10px;
+
+    h3 {
+      font-family: $font-family-base;
+      font-size: $font-size-h2;
+    }
+  }
+
+  .widget-content {
+    padding: 0;
+
+    table {
```

Comment:
is this table styled like any other table? Or is this style unique only to the dashboard page?

Reply:
What about this?

Reply:
This is removed, instead of this, we have some section specific css is there now. https://github.com/civicrm/org.civicrm.shoreditch/pull/35/commits/6a4b3ecbbe162088fca0be019b89ace50ee84f17#diff-5293c2ba2447824a77c8243b0ad694e0R93